docs/source/reference/api-server/api-server-admin-deploy.rst

```diff
@@ -289,10 +175,165 @@ Test the API server by curling the health endpoint:
+            You can use the ``exec_kubeconfig_converter.py`` script to do this.
+
+            .. code-block:: bash
+
+                python -m sky.utils.kubernetes.exec_kubeconfig_converter --input ~/.kube/config --output ~/.kube/config.converted
+
+            Then create the Kubernetes secret with the converted kubeconfig file ``~/.kube/config.converted``.
+
+        To use multiple Kubernetes clusters, you will need to add the context names to ``allowed_contexts`` in the SkyPilot config. An example config file that allows using the hosting Kubernetes cluster and two additional Kubernetes clusters is shown below:
+
+        .. code-block:: yaml
+
+            kubernetes:
+              allowed_contexts:
+              # The hosting Kubernetes cluster, you cannot set if the hosting is disabled by kubernetesCredentials.useApiServerCluster=false
+              - in-cluster
+              # The additional Kubernetes context names in the kubeconfig you configured
+              - context1
+              - context2
+
+        Refer to :ref:`sky-api-server-config` for how to set the SkyPilot config in Helm chart values.
+
+    .. tab-item:: AWS
+        :sync: aws-creds-tab
+
+        Make sure you have the access key id and secret access key.
+
+        Create a Kubernetes secret with your AWS credentials:
+
+        .. code-block:: bash
+
+            kubectl create secret generic aws-credentials \
+              --namespace $NAMESPACE \
+              --from-literal=aws_access_key_id=YOUR_ACCESS_KEY_ID \
+              --from-literal=aws_secret_access_key=YOUR_SECRET_ACCESS_KEY
+
+        Replace ``YOUR_ACCESS_KEY_ID`` and ``YOUR_SECRET_ACCESS_KEY`` with your actual AWS credentials.
+
+        Enable AWS credentials by setting ``awsCredentials.enabled=true`` and ``awsCredentials.awsSecretName=aws-credentials`` in the Helm values file.
+
+        .. code-block:: bash
+
+            helm upgrade --install skypilot skypilot/skypilot-nightly --devel \
+                --namespace $NAMESPACE \
+                # keep the Helm chart values set in the previous step
+                --reuse-values \
+                --set awsCredentials.enabled=true
+
+        .. dropdown:: Use existing AWS credentials
+
+            You can also set the following values to use a secret that already contains your AWS credentials:
+
+            .. code-block::bash
+                # TODO: replace with your secret name and keys in the secret
+                helm upgrade --install skypilot skypilot/skypilot-nightly --devel \
+                    --namespace $NAMESPACE \
+                    --reuse-values \
+                    --set awsCredentials.enabled=true \
+                    --set awsCredentials.awsSecretName=your_secret_name \
+                    --set awsCredentials.accessKeyIdKeyName=aws_access_key_id \
+                    --set awsCredentials.secretAccessKeyKeyName=aws_secret_access_key
```

Comment:
This somehow failed to render. Do we need an additional new line after `code-block`?

Reply:
Good catch